vofs/vofs/CadcCache.py

```diff
@@ -249,9 +250,8 @@ def open(self, path, isNew, mustExist, ioObject, trustMetaData):
                 # are propegated.
                 if not (isinstance(fileHandle.readException[1], EnvironmentError) and
                                 fileHandle.readException[1].errno == errno.ENOENT and not mustExist):
-                    raise fileHandle.readException[0], \
-                        fileHandle.readException[1], \
-                        fileHandle.readException[2]
+                    raise fileHandle.readException[0]('{} - {}'.format(
```

Comment:
readException is set by calling sys.exc_info() that returns that tuple - hence we expect that.